js/adapt-tincan.js

```diff
@@ -1,51 +1,77 @@
 /*
-* adapt-tincan
-* License - http://github.com/adaptlearning/adapt_framework/LICENSE
-* Maintainers - Dennis Heaney <dennis@learningpool.com>
-*/
+ * adapt-tincan
+ * License - http://github.com/adaptlearning/adapt_framework/LICENSE
+ * Maintainers  - Dennis Heaney <dennis@learningpool.com>
+ *              - Barry McKay <barry@learningpool.com>
+ *              - Andy Bell <andrewb@learningpool.com>
+ */
 define(function(require) {
 
   var Adapt = require('coreJS/adapt');
   var _ = require('underscore');
   var xapi = require('extensions/adapt-tincan/js/xapiwrapper.min');
+
   var xapiWrapper;
   var STATE_PROGRESS = 'adapt-course-progress';
 
   var TinCan = Backbone.Model.extend({
+    actor: null,
+    activityId: null,
 
     defaults: {
       initialised: false,
       state: null
     },
 
-    initialize: function () {
-      this.setConfig(Adapt.config.get('_tincan'));
+    initialize: function() {
+      if (window.xapiWrapper) {
+        xapiWrapper = window.xapiWrapper;
+      } else {
+        xapiWrapper = ADL.XAPIWrapper;
+      }
+
+      if (!Adapt.config.get("_extensions") || !Adapt.config.get("_extensions")._tincan) {
+        return;
```

Comment:
I can't be.  Have a look at this:
https://github.com/adaptlearning/adapt_authoring/blob/develop/lib/outputmanager.js#L143